pod-security-policy/metadata-restrictions/template.yaml

```diff
@@ -2,27 +2,39 @@ apiVersion: templates.gatekeeper.sh/v1beta1
 kind: ConstraintTemplate
 metadata:
   name: metadatarestrictions
+  annotations:
+    description: Places restrictions on the metadata of a k8s object
 spec:
   crd:
     spec:
       names:
         kind: MetadataRestrictions
       validation:
         openAPIV3Schema:
+          description: "Represents the schema of a k8s object that subscribes to the metadatarestrictions constraint template"
           type: object
-          properties:
-            annotations:
+          properties: 
+            annotations:  
```

Comment:
```suggestion
          properties:
            annotations:
```

Reply:
If you're using VSCode, I suggest you set it up to automatically chomp whitespace and to add newlines at the end of files. :)